riocli/jsonschema/schemas/static_route-schema.yaml

```
@@ -51,11 +51,18 @@ definitions:
   uuid:
     type: string
     pattern: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+  ipRange:
+    type: string
+    pattern: '^\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}/\d{1,2}$'
```

This should validate the IPv4 CIDR Block better, I think.
```suggestion
    pattern: '^((25[0-5]|(2[0-4]|1\d|[1-9]|)\d)\.?\b){4}/([1-9]|1\d|2\d|3[0-2])$'
```

---

This string does not work: `192.168.1.1/32`.
https://regex101.com/r/8gzy9m/1

---

It does. Your example has an additional quote at the end. Must be a typo while copying the pattern from the file.
https://regex101.com/r/32djN5/1